contrib/sarplus/DEVELOPMENT.md

```diff
@@ -39,3 +39,33 @@ sbt test
 ```
 
 (use ~test and it will automatically check for changes in source files, but not build.sbt)
+
```

Comment:
are the previous instructions still functional (e.g. sbt spPublish?)

Reply:
OK, let me try to update the instructions as well

Reply:
Updated.  Please take a look and leave comments if any improvements